src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -467,7 +467,58 @@ anticlockwise | anticlockwise
 
 #### PointOnRoute
 
-TBD
+For each `stop_point` of all the `vehicle_journey` of the current `route`, a
+`PointOnRoute` is created, with a link to a corresponding `RoutePoint` (see
+[related section](#routepoint)).
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+PointOnRoute/@id | | | see (1) below
+PointOnRoute/@version | | | fixed value `any`
+PointOnRoute/@order | | | see (2) below
+PointOnRoute/RoutePointRef/@ref | | | see (1) below; see [id formatting](#id-of-objects) with `RoutePoint` as object type for the rest
+
+**(1) identifier for PointOnRoute**
+
+The identifier is built from the concatenation of the following fields separated
+by `_`:
+- the `route_id`
+- the `@order`
+
+For the rest of the identifier, use [id formatting](#id-of-objects).
+
+**(2) `@order` of the PointOnRoute**
+
+All the `stop_point` of the `route` are inserted with the following rules
+(`@order` is auto-incremented from `1`).
+
+The `vehicle_journeys` are processed in by alphanumerical order of their
```

Comment:
alphanumerical to remove

Reply:
Done in ff10b51c119dd2c22fb21e441eaa4ea7088c0098.